packages/components/src/instance-initializers/load-sprite.ts

```diff
@@ -6,6 +6,7 @@
 import config from 'ember-get-config';
 
 export async function initialize() {
+  // SHOULD THIS BE REMOVED? OR SHOULD WE LOOK FOR A DIFF CONFIG KEY?
   // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
   if (config?.emberFlightIcons?.lazyEmbed) {
```

Comment:
Should this config be renamed? it is used in the showcase, atlas, and cloud-ui

Reply:
Also, if I remove it from the showcase - icons dont load in percy.

Reply:
This would be breaking change if you change this key, as it's being set on consumer side

Reply:
Good idea! I added a new config option called `flightIconsSpriteLazyEmbed` and I can open a ticket to deprecate the old config in v5.0.0.

Reply:
Ticket: https://hashicorp.atlassian.net/browse/HDS-4883?atlOrigin=eyJpIjoiZDM0NDM3M2JlNDFmNDFjMDg3NGQyOWU5NTdmM2U4NmEiLCJwIjoiaiJ9